### ISSUE TITLE
[macOS] ASCII art and entries aren't colored

### ISSUE THREAD

**Issue by GentleHoneyLover** [HorlogeSkynet/archey4#98]
**Describe the bug**
archey4 produces a correct output, however ascii art and categories are not colored.

**Expected behavior**
ascii art and categories are colored

**Screenshots**
archey4 output:
<img width="542" alt="Screen Shot 2021-04-19 at 16 27 58" src="https://user-images.githubusercontent.com/38346023/115253391-a1cafd80-a12c-11eb-888f-baaf7351a572.png">

original archey output:
<img width="546" alt="Screen Shot 2021-04-19 at 16 29 07" src="https://user-images.githubusercontent.com/38346023/115253481-b6a79100-a12c-11eb-9cc6-7a1cc6200e96.png">

**Environment**
 - Version used: 4.11
 - Method of installation: `brew install archey4 `
 - Hardware type: MacBook Pro (16-inch, 2019) 
 - Python version: 3.9.4
 - Operating system and version: macOS Big Sur 11.2.3 
 - Graphical environment name and version: Aqua
 - Connectivity: Internet access via Wi-Fi

**Comment by HorlogeSkynet** [comment]
Hi @GentleHoneyLover ! Thanks for opening this up.

I won't lie to you, it was mainly on purpose so as to keep a first support for Darwin very easy and quick to implement.
On the other hand, Apple logo isn't colored anymore since 1998... So I really wonder(ed) whether Archey should keep it that way.

Tell me what you think about that. Bye 👋

**Comment by GentleHoneyLover** [comment]
Hi @HorlogeSkynet,

I personally like the retro-style of apple logo in the original archey. Keeping it just white is a dull and not interesting. I would personally prefer archey4 to use exactly the same ascii art and coloring as original archey. It looks clean, minimalistic and classy (with a touch of retro).

I would also prefer the categories to be colored (User, Hostname, Model, etc) — otherwise it's very difficult to read the output!

I wish I could help with a pull request but I'm not a coder :(

But great project — thanks for doing this! I prefer archey to neofetch because it's fast and has no dependencies. It's great to see someone carrying it forward!

**Comment by HorlogeSkynet** [comment]
> I personally like the retro-style of apple logo in the original archey.

Well, I guess we'll have to go through a configurable option then. I'll try to think about a _KISS_ implementation to allow logo variants.

> Keeping it just white [...].

Actually, it's a different white as the one use for the text 😉

> I would also prefer the categories to be colored [...] !

Do you think about a specific color for them ?

> I wish I could help with a pull request but I'm not a coder :(

It's not an issue. Would you be able to checkout a Git branch and locally trying it out though ?

> But great project — thanks for doing this! I prefer archey to neofetch because it's fast and has no dependencies. It's great to see someone carrying it forward!

Thanks, appreciated 🙇
There are actually 2 dependencies, including one optional. Anyway, if you install from with `pip` or `brew`, they are "implicit" from an user PoV.

Bye 👋

**Comment by GentleHoneyLover** [comment]
> Do you think about a specific color for them ?
Cyan (like in archey) works well I think. I know that neofetch, for example, has an accent color configurable via config file. Since you already have a config file implemented, maybe there could be an option switch to customize the default accent color?

> It's not an issue. Would you be able to checkout a Git branch and locally trying it out though ?
Sure, that I can do. Let me know when you do changes you'd like me to test

> There are actually 2 dependencies, including one optional. Anyway, if you install from with pip or brew, they are "implicit" from an user PoV.
Sure, it's nothing like neofetch which pulls two dozed of dependencies with it.

**Comment by GentleHoneyLover** [comment]
@HorlogeSkynet,

Thanks for the update! Here's what I'm getting:

<img width="535" alt="Screen Shot 2021-04-27 at 20 40 36" src="https://user-images.githubusercontent.com/38346023/116295015-e7c13a80-a798-11eb-8259-11b167a7fbef.png">

Looks much cooler! ;) 

Do you think it would be possible to update the ascii art itself? Using just "#" symbol looks much cleaner :) But I'm being picky I guess.

                 ###
               ####
               ###
       #######    #######
     ######################
    #####################
    ####################
    ####################
    ##################### 
     ######################
      ####################
        ################
         ####     #####

Another piece of feedback is that you might want to consider "proper" human-readable names for the categories. E.g., "Desktop Environment:" instead of "DesktopEnvironment:" or "LAN IP:" instead of "LanIP:", etc. 

Thanks for the great project! ;)

**Comment by HorlogeSkynet** [comment]
> Another piece of feedback is that you might want to consider "proper" human-readable names for the categories. E.g., "Desktop Environment:" instead of "DesktopEnvironment:" or "LAN IP:" instead of "LanIP:", etc.

I'm pinning this to [Core](https://github.com/HorlogeSkynet/archey4/projects/3#card-60041643) project kanban. I'll try to handle it soon, before drafting a new version.

**Comment by HorlogeSkynet** [comment]
> Another piece of feedback is that you might want to consider "proper" human-readable names for the categories. E.g., "Desktop Environment:" instead of "DesktopEnvironment:" or "LAN IP:" instead of "LanIP:", etc.

... and I almost forgot, you can already "customize" entries names from configuration, with the `name` key. From README :

```javascript
// ...
		{
			"type": "CPU",
			// Set to `true` to join all CPUs on the same line.
			"one_line": false,
			// Set to `false` to hide the number of cores.
			"show_cores": true,
			//
			// As explained above, you may rename entries as you wish.
			"name": "Processor"
		},
// ...
```

**Comment by GentleHoneyLover** [comment]
Got it! Thanks!

**Comment by HorlogeSkynet** [comment]
Both features have been released in [v4.12.0](https://github.com/HorlogeSkynet/archey4/releases/tag/v4.12.0), and already available on Homebrew. Cheers 👋
